faq/dump-css.md

```diff
@@ -0,0 +1,10 @@
+## What is CSS?
+
+Before diving in to how best to dump a CSS protected disc in Aaru, you should first have a decent understand of what CSS even is. Basically, it's a standardized and widely used form of encryption for movie DVDs designed to prevent unlicensed players from playing the DVDs, as well as to prevent copies being made of the discs. Sectors encrypted by CSS are marked as such, and the disc and drive have to authenticate one another before playback can start. Not every DVD player can play ever CSS encrypted disc either, as one aspect of CSS encryption is region locking. Basically this means that if the disc is only licensed to play in a certain region, the authentication process will fail if the drive reading it isn't set to that region. Most PC drives do allow you to change which region they're able to read a set number of times, but most standalone DVD players do not. This is purely a form of copy protection, and not a physical or digital difference in encoding, unlike other technologies (such as VHS) that were physically unable to play media from other regions. Every encrypted sector can be decrypted with a so-called title key, and the vast majority of the time, this title key will be the same for every sector on the disc. 
```

Comment:
decent understandING

Reply:
"widely used"? it's THE DVD Video standard encryption, not any other

Reply:
the title key is usually the same for every DVD Title (aka chapter), but that's not always the case, and is certainly not "majorly" the same for all a disc